pkgs/by-name/ef/efibooteditor/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  qt6,
+  zlib,
+  cmake,
+  efivar,
+  pkg-config,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "efibooteditor";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "Neverous";
+    repo = "efibooteditor";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-xD40ZzkpwerDYC8nzGVqEHLV0KWbxcc0ApquQjrPJTc=";
+  };
+
+  buildInputs = [ zlib ] ++ lib.optional stdenv.hostPlatform.isLinux efivar;
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    qt6.qttools
+    qt6.wrapQtAppsHook
+  ];
+
+  postPatch = ''
+    substituteInPlace misc/EFIBootEditor.desktop \
+      --replace-fail "1.0" ${finalAttrs.version}
+  '';
+
+  env.BUILD_VERSION = "v${finalAttrs.version}";
+  cmakeBuildType = "MinSizeRel";
+  cmakeFlags = [ "-DQT_VERSION_MAJOR=6" ];
+
+  postInstall = ''
+    install -Dm644 $src/LICENSE.txt $out/share/licenses/${finalAttrs.pname}/LICENSE
+  '';
+
+  meta = {
+    description = "Boot Editor for (U)EFI based systems";
+    homepage = "https://github.com/Neverous/efibooteditor";
+    changelog = "https://github.com/Neverous/efibooteditor/releases/tag/v${finalAttrs.version}";
+    license = lib.licenses.lgpl3Plus;
+    broken = stdenv.hostPlatform.isDarwin; # TODO broken on darwin
+    platforms = lib.platforms.linux;
```

Comment:
should I remove this broken line and let it be `platforms = lib.platforms.linux;`? I don't have the access to fix the darwin build.